docs/your-first-script.md

```diff
@@ -2,115 +2,226 @@
 
 # Your first script
 
+Once you have Nextflow installed, you’re ready to run your first script. This guide details how to run a basic Nextflow pipeline. It includes:
+
+- Running a pipeline
+- Modifying and resuming a pipeline
+- Configuring a pipeline parameter
+
+**Prerequisites**
+
+You will need the following to get started:
+
+- Nextflow 24.10.0 or later. See {ref}`install-page` for instructions to install or update your version of Nextflow.
+
+:::{note}
+This guide uses the second preview of publish targets, introduced in Nextflow 24.10.0. Publish targets are intended to replace the `publishDir` directive.
```

Comment:
Think it should not be used a feature under dev in the getting started guide

Reply:
Learning a new tool and starting a getting warning is evil. I think we should use document what's the current approach that's `publishDir`

Reply:
Fair point - I've removed the targets and added publishDir.